core/contracts/financial-templates/implementation/PricelessPositionManager.sol

```diff
@@ -75,11 +77,8 @@ contract PricelessPositionManager is FeePayer {
     event NewSponsor(address indexed sponsor);
     event Redeem(address indexed sponsor, uint indexed collateralAmount, uint indexed tokenAmount);
     event ContractExpired(address indexed caller);
-    event SettleExpiredPosition(
-        address indexed caller,
-        uint indexed collateralAmountReturned,
-        uint indexed tokensBurned
-    );
+    event SettleExpiredPosition(address indexed caller, uint indexed collateralReturned, uint indexed tokensBurned);
```

Comment:
This refactor I did to clean up the variable names in the `SettleExpiredPosition` event.